chainerx_cc/chainerx/backward_test.cc

```diff
@@ -334,15 +334,29 @@ TEST_F(BackpropTest, BackwardWithComplicatedRanks) {
     });
 }
 
-TEST_F(BackpropTest, TryBackwardFromArrayWithoutNode) {
+TEST_F(BackpropTest, BackwardFromArrayWithoutNode) {
     auto xs = MakeFullArrays({1}, {2.0f, 3.0f});
     auto y1 = xs[0] * xs[1];  // without graph
-    EXPECT_THROW(Backward(y1), ChainerxError);
+    Backward(y1);
+
+    EXPECT_THROW(xs[0].GetGrad(), ChainerxError);
+    EXPECT_THROW(xs[1].GetGrad(), ChainerxError);
+}
+
+TEST_F(BackpropTest, BackwardFromArrayWithAndWithoutNode) {
```

Comment:
Are the comments in this test reversed ("with"/"without")?

Reply:
Oops, I thought the comments were wrong, but I was wrong.